### PR TITLE
Features/button component

### PR DESCRIPTION
**Pull request by Wannacryy24**

### CODE REVIEW

File: src/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React, { Children } from 'react'
+
+export default function Button({Children,className,disabled}) {
```
Comment (Sujal0007) [inline src/Button.jsx]: we can also add the styles for button like height , width and size .

Comment (Wannacryy24) [reply src/Button.jsx]: yes i am on it now, it was just a demo right now

Comment (Sujal0007) [inline src/Button.jsx]: Event handlers are also missing like onClick.

File: src/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './Button.css'
+export default function Button(props) {
+   const {
```
Comment (Sujal0007) [inline src/Button.jsx]: Can also add diffrent variants using conditionals

File: src/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    onClick,
+    variant='',
+    height='auto',
+    width='auto',
+    size='16px',
+    onMouseEnter,
+    onMouseLeave,
+    img,
+    src
+   } = props;
```
Comment (Sujal0007) [inline src/Button.jsx]: Can also provide some default values to props

File: src/Button.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  const buttonClass = `${className} ${variant?variant:""}`
```
Comment (Sujal0007) [inline src/Button.jsx]: can add default values in props to these or cover the edge case if these are not defined

File: src/Button.jsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  const styleProps = {
+    backgroundColor:color,
+    height:height,
+    width:width,
+    fontSize:size
+  };
+
+  let buttonClass = `${className} ${variant?variant:""} ${color ? `btn-${color}` : ''}`;
+  
+  switch(variant){
```
Comment (Sujal0007) [inline src/Button.jsx]: default case is missing which can handle undefined variants

**Pull request merged**